Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -192,3 +192,27 @@ public struct Aside {
         self.content = Array(newBlockQuote.blockChildren)
     }
 }
+
+extension BlockQuote {
+    /// Conservatively checks the text of this block quote to see whether it can be parsed as an aside.
+    ///
+    /// Whereas ``Aside/init(_:)`` will use all the text before the first colon in the first line,
+    /// or else return an ``Aside`` with a ``Aside/Kind-swift.struct`` of ``Aside/Kind/note``,
+    /// this function will allow parsers to only parse an aside if there is a single-word aside
+    /// marker in the first line, and otherwise fall back to a plain ``BlockQuote``.
+    func isAside() -> Bool {
```

Comment:
```suggestion
    var isAside: Bool {
```

Should this better be a computed property instead of being a func?

Reply:
IMO this is fine as a function because it indicates to the caller that it shouldn't assume an O(1) operation, in this case because of the implementation loops over the block quotes's children.